samples/WinForms/Dx12Panel.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+using System;
+using System.Windows.Forms;
+using TerraFX.Samples.DirectX.D3D12;
+namespace TerraFX.Samples.WinForms
+{
+    public class DX12Panel : Panel
+    {
+        private readonly DX12Viewport _dxVp;
```

Comment:
It would be nice to just name the field `_viewport`, it will help with readability of the code

Reply:
agreed.